requirements/requirements_testing.in

```diff
@@ -0,0 +1,5 @@
+# NOTE: Only dependencies needed to use ansible_base.lib.testing.* should go here.
+# Deps specific to django-ansible-base tests should go in requirements_dev.txt
+cryptography
+pytest<8  # https://github.com/pytest-dev/pytest/issues/11868
+pytest-django
```

Comment:
@AlanCoding well the issue is: If I don't pin to a version here (and only pin it in the DAB-specific `requirements_dev.txt`), the updater script will stick 8.0.0 in `requirements_all.txt`. And then when tox creates the test env, it will install from `requirements_all.txt` *and* `requirements_dev.txt` and pip will error because it doesn't know if we want 8 or < 8.

It's an annoying situation.

Reply:
That looks kind of like an unrelated CI fix.

Reply:
I think we can drop the pin now with pytest-asyncio 0.23.4 out, it has a proper constraint on pytest now.